charts/nm-vllm-grafana-dashboards/templates/config-map-list.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMapList
+items:
+{{- $files := .Files.Glob "dashboards/*.json" }}
+{{- range $path, $fileContents := $files }}
+{{- $dashboardName := regexReplaceAll "(^.*/)(.*)\\.json$" $path "${2}" }}
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: {{ template "nm-vllm-grafana-dashboards.fullname" $ }}-grafana-dashboard
```

Comment:
Fun fact, `range` changes the scope, so this was breaking for me when it was 
```
 name: {{ template "nm-vllm-grafana-dashboards.fullname" . }}-grafana-dashboard
```
So I changed the passed scope from `.` to `$` instead. That is what both the [`grafana`](https://github.com/grafana/helm-charts/blob/main/charts/grafana/templates/dashboards-json-configmap.yaml#L7) and [`kube-prometheus-stack`](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/grafana/configmap-dashboards.yaml#L12) charts do in this same situation.

Reply:
Oh interesting, and you are making me realize I must have never tested this change 🤦‍♂️